irma/cmd/config.go

```diff
@@ -0,0 +1,202 @@
+package cmd
+
+import (
+	"crypto/tls"
+	"net/smtp"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/go-errors/errors"
+	"github.com/mitchellh/mapstructure"
+	irma "github.com/privacybydesign/irmago"
+	"github.com/privacybydesign/irmago/server"
+	"github.com/privacybydesign/irmago/server/keyshare"
+	"github.com/spf13/cast"
+
+	"github.com/sietseringers/cobra"
```

Comment:
Because some time ago I added to `cobra` the headers that you see in the output of `irma server -h`, and now also in the help messages of the various keyshare commands. `viper` then also needed to be forked to get it to depend on our forked `cobra.` At some point we can try to get that merged into upstream, or define a custom help command and stop depending on these forks.